lib/chat_api_web/controllers/note_controller.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ChatApiWeb.NoteController do
+  use ChatApiWeb, :controller
+
+  alias ChatApi.Notes
+  alias ChatApi.Notes.Note
+
+  action_fallback ChatApiWeb.FallbackController
+
+  @spec index(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def index(conn, _params) do
+    notes = Notes.list_notes()
+    render(conn, "index.json", notes: notes)
+  end
+
+  @spec create(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def create(conn, %{"note" => note_params}) do
+    with {:ok, %Note{} = note} <- Notes.create_note(note_params) do
```

Comment:
let's make sure when we're creating a new `note`, we don't allow the client to specify the `account_id` or the `author_id` -- these should come directly from `conn.assigns.current_user`